tests/subsys/bootloader/flash_patch/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Is this script pointed to somewhere? If not, I fear for code rot. Either it should be executed by CI, or we should point to it in some documentation.

Reply:
ping @oyvindronningstad , would be good to clarify this before merging.

Reply:
I'll update docs

Reply:
Turns out we don't have detailed docs for the immutable bootloader, so I don't know where to put this.

Reply:
This will be hooked up to CI.